scripts/build-image

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Manually, when you want to build a container after a release for now. Will need a release script at some point. It is not terribly useful at the moment.